src/api/graphql.ts

```diff
@@ -1,52 +1,27 @@
 import type { GatsbyFunctionRequest, GatsbyFunctionResponse } from 'gatsby'
-import { GraphQLError } from 'graphql'
 
-import type { FormatError } from '../server/envelop'
-import { getEnvelop } from '../server/envelop'
-import persisted from '../../@generated/graphql/persisted.json'
-
-const persistedQueries = new Map(Object.entries(persisted))
+import { execute } from '../server'
 
 const parseRequest = (req: GatsbyFunctionRequest) => {
-  const { operationName, variables } =
+  const { operationName, variables, query } =
     req.method === 'POST'
       ? req.body
       : {
           operationName: req.query.operationName,
-          variables: JSON.parse(req.query.variables),
+          variables: JSON.parse(req.query.variables?.toString()),
```

Comment:
```suggestion
          variables: req.query.variables?,
```

Why do you need to transform to string to parse to JSON?

Reply:
I think we don't need it. Nice!